include/dd/Control.hpp

```diff
@@ -12,8 +12,8 @@
 
 namespace dd {
     struct Control {
-        enum class Type : bool { pos = true,
-                                 neg = false };
+        enum class Type : bool { pos = true,    // NOLINT(readability-identifier-naming)
+                                 neg = false }; // NOLINT(readability-identifier-naming)
```

Comment:
Would it be too inconvenient to change these? I would assume they are not used in that many places.

Reply:
It's more about usage outside of the package. As we expose this towards the public I'm slightly in favor of keeping the naming as is.

Reply:
True. Would certainly be a breaking change. The question is: how many people are affected by that change and how hard is it to adapt. I would argue that changing capitalization of a tiny struct's members wouldn't be too much of a hassle. In the end, I'll just leave it to you to decide. I am fine with either solution.

Reply:
As discussed, we'll keep it as proposed. Future plan will probably necessitate changes here anyway and we don't need to break things twice.